src/Controls/src/Core/Brush.cs

```diff
@@ -19,146 +19,146 @@ public static bool IsNullOrEmpty(Brush brush)
+		public static SolidColorBrush Coral => new(Colors.Coral);
+		public static SolidColorBrush CornflowerBlue => new(Colors.CornflowerBlue);
+		public static SolidColorBrush Cornsilk => new(Colors.Cornsilk);
+		public static SolidColorBrush Crimson => new(Colors.Crimson);
+		public static SolidColorBrush Cyan => new(Colors.Cyan);
+		public static SolidColorBrush DarkBlue => new(Colors.DarkBlue);
+		public static SolidColorBrush DarkCyan => new(Colors.DarkCyan);
+		public static SolidColorBrush DarkGoldenrod => new(Colors.DarkGoldenrod);
+		public static SolidColorBrush DarkGray => new(Colors.DarkGray);
+		public static SolidColorBrush DarkGreen => new(Colors.DarkGreen);
+		public static SolidColorBrush DarkKhaki => new(Colors.DarkKhaki);
+		public static SolidColorBrush DarkMagenta => new(Colors.DarkMagenta);
+		public static SolidColorBrush DarkOliveGreen => new(Colors.DarkOliveGreen);
+		public static SolidColorBrush DarkOrange => new(Colors.DarkOrange);
+		public static SolidColorBrush DarkOrchid => new(Colors.DarkOrchid);
+		public static SolidColorBrush DarkRed => new(Colors.DarkRed);
+		public static SolidColorBrush DarkSalmon => new(Colors.DarkSalmon);
+		public static SolidColorBrush DarkSeaGreen => new(Colors.DarkSeaGreen);
+		public static SolidColorBrush DarkSlateBlue => new(Colors.DarkSlateBlue);
+		public static SolidColorBrush DarkSlateGray => new(Colors.DarkSlateGray);
+		public static SolidColorBrush DarkTurquoise => new(Colors.DarkTurquoise);
+		public static SolidColorBrush DarkViolet => new(Colors.DarkViolet);
+		public static SolidColorBrush DeepPink => new(Colors.DeepPink);
+		public static SolidColorBrush DeepSkyBlue => new(Colors.DeepSkyBlue);
+		public static SolidColorBrush DimGray => new(Colors.DimGray);
+		public static SolidColorBrush DodgerBlue => new(Colors.DodgerBlue);
+		public static SolidColorBrush Firebrick => new(Colors.Firebrick);
+		public static SolidColorBrush FloralWhite => new(Colors.FloralWhite);
+		public static SolidColorBrush ForestGreen => new(Colors.ForestGreen);
+		public static SolidColorBrush Fuchsia => new(Colors.Fuchsia);
+		public static SolidColorBrush Gainsboro => new(Colors.Gainsboro);
+		public static SolidColorBrush GhostWhite => new(Colors.GhostWhite);
+		public static SolidColorBrush Gold => new(Colors.Gold);
+		public static SolidColorBrush Goldenrod => new(Colors.Goldenrod);
+		public static SolidColorBrush Gray => new(Colors.Gray);
+		public static SolidColorBrush Green => new(Colors.Green);
+		public static SolidColorBrush GreenYellow => new(Colors.GreenYellow);
+		public static SolidColorBrush Honeydew => new(Colors.Honeydew);
+		public static SolidColorBrush HotPink => new(Colors.HotPink);
+		public static SolidColorBrush IndianRed => new(Colors.IndianRed);
+		public static SolidColorBrush Indigo => new(Colors.Indigo);
+		public static SolidColorBrush Ivory => new(Colors.Ivory);
+		public static SolidColorBrush Khaki => new(Colors.Ivory);
```

Comment:
If I see typos like this, it's probably not on purpose?

Reply:
I found 3 like this after adding a test to check for it.

Reply:
yea, definitely not on purpose but it's been like that since XF :-/ 

https://github.com/xamarin/Xamarin.Forms/blob/ba709a180f1ac2218a1c9fa8b0566bd92e96e215/Xamarin.Forms.Core/Brush.cs#L78

So probably been wrong for a bit now